extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -498,6 +498,7 @@ describe('amp-analytics.visibility', () => {
       unobserveSpy = sandbox.stub();
       callbackSpy1 = sandbox.stub();
       callbackSpy2 = sandbox.stub();
+      ampdoc.win.IntersectionObserver = {};
```

Comment:
Good catch, skipping if no `ampdoc.win.IntersectionObserver`, can't find a way to stub non-existing methods with sinon.